src/models/model.py

```diff
@@ -111,20 +107,32 @@ def validation_step(self, batch, batch_idx):
             
         # log FID
         if batch_idx % self.fid_freq == 0:
-            self._log_fid('valid', pred, target)
+            self._log_fid(pred, target, step_name='valid')
 
         # metrics
         ssim_val = ssim(pred, target)
         psnr_val = psnr(pred, target)
         self.log('valid_ssim', ssim_val)
         self.log('valid_psnr', psnr_val)
 
+    def test_step(self, batch, batch_idx):
```

Comment:
at the end of training I run a "test step" but its just convenient for predicting on the val images I've kept aside to do this